crossbar/router/broker.py

```diff
@@ -178,13 +179,19 @@ def processPublish(self, session, publish):
         if store_event:
             self.log.debug('Persisting event on topic "{topic}"', topic=publish.topic)
 
+        # check if the event is to be retained by inspecting the 'retain' flag
+        retain_event = False
+        if getattr(publish, "retain", False):
```

Comment:
lol the tests fail anyway because i did `subscribe.get_retained`
how do we work around this? release an AB?